src/utils/VirtualModule.ts

```diff
@@ -30,6 +30,22 @@ function getNodeModulesDir() {
   return cachedNodeModulesDir;
 }
 
+function getSuffix(name: string): string {
```

Comment:
It looks great to me, can you cover this function with a test with few different test cases?